src/controller/CHIPDeviceController.cpp

```diff
@@ -2018,9 +2063,13 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         // Read the requested minimum connection times from all network commissioning clusters
         readPaths[7] = app::AttributePathParams(app::Clusters::NetworkCommissioning::Id,
                                                 app::Clusters::NetworkCommissioning::Attributes::ConnectMaxTimeSeconds::Id);
+        // Read the current fabrics
+        readPaths[8] = app::AttributePathParams(app::Clusters::OperationalCredentials::Id,
+                                                app::Clusters::OperationalCredentials::Attributes::Fabrics::Id);
```

Comment:
Doing this as a required read for all commissioning interactions just to support commissioners that might use this state is not great: this attribute can be quite a lot of data, no?

Why does this need to be done in the common code at all, instead of the commissioners that want this doing the relevant read themselves over PASE before the start the common commissioning process?

Reply:
updated to make this attribute read runtime configurable (disabled by default)